### PR TITLE
[CO-116] Bind Amex Events to Signal

### PR DESCRIPTION
**Pull request by vanessagl2**

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "saks-signal",
-  "version": "0.1.23",
+  "version": "0.1.24",
```
Comment (mikeumus) [inline package-lock.json]: Was this modified by hand? I think when we have a version change the checksum also might change.

File: dist/site-catalyst.js [hunk 2]
```diff
@@ -1823,6 +1830,16 @@ module.exports = function (app) {
           app.events = app.apl(app.events, 'event95', ',', 2);
         }
 
+        if (amex_redeem_amount) {
+          thisProduct[4] += (thisProduct[4] ? '|' : '') + 'event123=' + amex_redeem_amount;
+          app.events = app.apl(app.events, 'event123', ',', 2);
+        }
+
+        if (amex_redeem_dollar) {
+          thisProduct[4] += (thisProduct[4] ? '|' : '') + 'event124=' + amex_redeem_dollar.toFixed(2);
```
Comment (mikeumus) [inline dist/site-catalyst.js]: Template literals are preferred but this isn't a blocker.

File: package-lock.json (hunk 1 above)
Comment (vanessagl2) [reply package-lock.json]: This was modified by `npm run release`

File: dist/site-catalyst.js (hunk 2 above)
Comment (vanessagl2) [reply dist/site-catalyst.js]: this package is auto-generated by `npm run release`

**Pull request merged**